tools/delete_candidate.php

```diff
@@ -142,12 +142,22 @@ function deleteCandidate($CandID, $PSCID, $confirm, $printToSQL, $DB, $output)
     if (is_null($sessions) || empty($sessions)) {
         echo "There are no corresponding sessions for CandID: $CandID \n";
     } else {
+        if ($outputType = "tosql") {
+            $subOutputType = "";
+        } else {
+            $subOutputType = $outputType;
+        }
```

Comment:
for lines 145-149, were the assignments supposed to be reversed? if they're reversed, the same issue as before persists - one SQL file is generated per session to be deleted. As is, the single SQL file that's generated only includes delete statements for candidate tables: 

![image](https://user-images.githubusercontent.com/17363738/34366934-e3c37170-ea70-11e7-96ce-866d59ddc184.png)